Core/HLE/sceDisplay.cpp

```diff
@@ -503,6 +503,17 @@ static void DoFrameDropLogging(float scaledTimestep) {
 	}
 }
 
+static int CalculateFrameSkip() {
+	// Used to calculate the FPS to skip if the user sets the type to percent of fps
```

Comment:
This comment should be moved into the right clause of the if statement below.

Besides, does this mean that since you can only set it up to 8, frameskip% can only be up to 8%?

Reply:
Sorry I missed that this was already discussed above. But what's the point of such a low frameskip percentage limit?  Shouldn't we change the scale entirely when the type is set to percentage?